src/Request.js

```diff
@@ -50,6 +50,19 @@ export default class Request {
             .join('&');
     }
 
+    static format_payload (payload, content_type) {
+        if (typeof payload !== 'object') {
```

Comment:
How I see it, if `payload` is a string, then we can assume that the payload has already been formatted to a string. This function is only supposed to format object payloads.

Do you have suggestions?

Reply:
Hmmm, if payload is a string, then we will just return the payload. So if it receives a `id=1996&month=2018-07-01` then it will just send that back. I dont think we need to wrap it in `""`.

I think I'm missing what you're trying to tell me. Can you provide an example?

Reply:
I meant that's what would happen from the old code, so we're changing the behavior of string is being handled when its assigned as payload.

ex:
```
this.auto_format = true;
this.headers['Content-Type'] = null;
this.data = 'test';

// Old code
payload = (!this.headers['Content-Type'] && this.data) ? Request.stringify(this.data) : JSON.stringify(this.data);

payload == ""test""
```